packages/browser/src/tracing/browserTracingIntegration.ts

```diff
@@ -355,11 +355,21 @@ export const browserTracingIntegration = ((_options: Partial<BrowserTracingOptio
           sampled: spanIsSampled(idleSpan),
           dsc: getDynamicSamplingContextFromSpan(span),
         });
+
+        scope.setSDKProcessingMetadata({
+          normalizedRequest: undefined,
+        });
       },
     });
 
     setActiveIdleSpan(client, idleSpan);
 
+    // We store the normalized request data on the scope, so we get the request data at time of span creation
+    // otherwise, the URL etc. may already be of the following navigation, and we'd report the wrong URL
+    getCurrentScope().setSDKProcessingMetadata({
+      normalizedRequest: getHttpRequestData(),
+    });
+
```

Comment:
m/q: maybe I'm missing some timing information but isn't there a good chance that setting the `normalizedRequest` to `undefined` in 360 applies to the same scope where we previously set it (370)?

Reply:
argh, actually that does not work I just noticed, it fixed one test but broke another one - as this resets the request before we even process the transaction 😬 need to do this in a different way... basically the problem was apparent in some e2e tests. I think it becomes problematic in certain cases, e.g. nextjs pages router, where routing instrumentation triggers navigation spans before the url is updated 🤔